c/misra/test/rules/RULE-2-8/test.c

```diff
@@ -0,0 +1,113 @@
+// Not a definition, only a declaration:
```

Comment:
Could you add some examples of `const` variables to this file?

For C++ one of the big problems with unused variable queries has been related to the value of `constexpr` variables being inlined in our AST, appearing to make such variables unused, when they are not. I don't think a similar problem exists with `const` variables, but it would be great to have a test case or two to (a) validate that and (b) protect against any future changes to the AST.